### DISCUSSION TITLE
Can't Get firecard set up properly

### DISCUSSION THREAD

**Discussion by jclaydon1969** [PRProd/HA-Firemote#263] [Q&A] [answered]
I have no clue what I'm doing wrong.
I downloaded the firemote card through HACS, installed the integration (i think) and then tried to register the resource.
So I googled that, and after clicking on the link, it couldn’t find anything… After a little more digging, I manages to find the relivant github folders in /homeassistant/www/community/HA-Firemote/

and tried typing this in the the previous link. Nothing. I then tried a direct link to the .json file, still nothing.

Then after more googling, I found this suggestion of adding this tho the configure.yaml so I copied it and changed it to this:

 resources:
    - url: /local/www/community/HA-Firemote/firemote.js
      type: module

which produced the following error:

duplicated mapping key (15:7)

 12 | resources:
 13 |     - url: /homeassistant/www/communi ...
 14 |       type: module

Does that mean that it’s already registered? I can’t figure out what I am doing wrong and why the card that I installed along with this doesn’t work because when I installed the card to a dashboard, nothing happens, it's completely bland and doesn't do anything

**Comment by PRProd** [comment]
Greetings @jclaydon1969!

Typically these kinds of strange things are related to your Home Assistant host needing a restart, your browser needing a refresh, or your companion app needing a cache refresh.  These things usually solve those kinds of problems.

Since I'm not sure how you are accessing your dashboards, (mobile, web, etc.) here are a couple of reading suggestions for you from other discussion threads:
 - [Tips on refreshing your dashboard view](https://github.com/PRProd/HA-Firemote/discussions/74#discussioncomment-5618842)
 - [Need help getting Firemote cards to show on my HA #242](https://github.com/PRProd/HA-Firemote/discussions/242)

I hope that these previous answers help.  If not, just let me know and we will get this sorted out for you!

-Doug

**Comment by jclaydon1969** [comment]
> Greetings @jclaydon1969!
> 
> Typically these kinds of strange things are related to your Home Assistant host needing a restart, your browser needing a refresh, or your companion app needing a cache refresh. These things usually solve those kinds of problems.
> 
> Since I'm not sure how you are accessing your dashboards, (mobile, web, etc.) here are a couple of reading suggestions for you from other discussion threads:
> 
>     * [Tips on refreshing your dashboard view](https://github.com/PRProd/HA-Firemote/discussions/74#discussioncomment-5618842)
> 
>     * [Need help getting Firemote cards to show on my HA #242](https://github.com/PRProd/HA-Firemote/discussions/242)
> 
> 
> I hope that these previous answers help. If not, just let me know and we will get this sorted out for you!
> 
> -Doug

Thank you for the reply..  Typically I am using firefox in linux, not sure if they have chrome.. took a look at other thread.  There was no developer option, but there was something called a web developer option, but it wasn't installed and when I tried to install the extention it said something about creating an account.

the home assistant sever is in virtualbox 7 on linux...

My windows machine went blooey ages ago.. But I have a laptop i can dig out of the archives....  

I tried the android home assistant app option, but I don't see any debugging option.


I have tried re-installing HACS, re-downloading the inregration library..  and I have tried installing it manually..  the results is always the same..    Restarting the entrire server,  clearing the entire cache browser

keep in mind, due to brain injuries from a severe lack of oxygen, I don't do very well with written instructions. I am more of a visual/audio learner..  Of course it doesn't help that I pretty much started learning this stuff a week ago.. lol
![firemote](https://github.com/PRProd/HA-Firemote/assets/43157088/f0dad238-fa22-471a-8776-390ac6ad2e94)


result is always the same.

picture attached.

**Reply by PRProd** [reply]
@jclaydon1969,

Oh man... brain injuries are tough.  It could be difficult for me to give you written instructions without giving you written instructions, but maybe if we take it very slow, we can get there.

I haven't made a Youtube video for myself, but I just found this one by a German poster: https://www.youtube.com/watch?v=GdFiCbyN4Lk - If you turn on the Closed Captions, you can also have them automatically translated into your own language which you might find helpful.

Let me know how it works for you.

**Comment by jclaydon1969** [comment, marked as answer]
update: the card isn't even installed because i tried your suggestion in the other thread you linked, and that's what the error was..  Going to try re-installing HACS and card again..  *sigh*

**Reply by PRProd** [reply]
Glad you found the issue!  Keep me posted!

**Reply by jclaydon1969** [reply]
Ok, I'm not entirely sure why, but for some reason I had to:
restore to a previous version of HA
re-install HACS
refresh.. 
re-install card
Refresh.
Click on the option that forces ALL data to be cleared when you exit browser (and yes, I did do this step several times before)
log back into HA and the card was there,  now I just have to figure out how to get it to switch inputs, I'm sure that's in the instructions somewhere.   For the record, pictures work fine, it just takes me longer..

Think of it this way, my brain doesn't work in straight lines..  it still works but it takes longer..  It's kind of like having to fly from the west coast all the way to the east coast, but you have to fly to China first..  You'll still get there, it just takes a long ass time and you're all tired and cranky when you finally do!  lol

Thanks for being patient.
